packages/task/src/browser/task-frontend-contribution.ts

```diff
@@ -137,6 +142,23 @@ export class TaskFrontendContribution implements CommandContribution, MenuContri
                 }
             }
         );
+        registry.registerCommand(
+            TaskCommands.TASK_CUSTOM_RUN,
+            {
+                isEnabled: () => true,
+                // tslint:disable-next-line:no-any
+                execute: (...args: any[]) => {
+                    const [source, label, overrides] = args;
```

Comment:
Why introducing new command `task:custom-run`?
Maybe just add an optional `overrides` parameter to the existing command `task:run`?

Reply:
I have thought about it and my first draft used that approach. But then I changed it for another command to not to mess things up. But I still in doubts what's better. Do you have some concerns or thought about advantages of one approach over the other?

Reply:
I'd choose the approach with a single command because of no code duplication at least.
I don't see any confusion with it. The idea is clear enough, as for me.

Reply:
Done